docker-compose.override.yml

```
@@ -20,3 +22,7 @@ services:
       - MYSQL_USER=coach
       - MYSQL_PASSWORD
       - MYSQL_ROOT_PASSWORD
+  cqfruler:
+    image: alphora/cqf-ruler:0.5.0
+    ports:
+      - "8080:8080"
```

Do you want to persist the database as well?

```suggestion
      - "8080:8080"
    volumes:
      - ./docker_data/cqfruler:/data
    environment:
      - SPRING_DATASOURCE_URL=jdbc:hsqldb:file:/data/h2
```